src/rgw/rgw_common.cc

```diff
@@ -1053,12 +1053,12 @@ Effect eval_or_pass(const boost::optional<Policy>& policy,
 		    const rgw::IAM::Environment& env,
 		    boost::optional<const rgw::auth::Identity&> id,
 		    const uint64_t op,
-		    const ARN& arn,
```

Comment:
why call it res?

Reply:
else where in a lot of coded "res" might stand for "result"--is there another word that can be resource?